.github/workflows/cd.yml

```diff
@@ -254,6 +301,39 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+  pypi-sdist:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Move pyproject.toml to the right location
+        run: cp pypi/pyproject.toml .
+      - name: Build sdist
+        uses: PyO3/maturin-action@v1
+        with:
+          command: sdist
+          args: --manifest-path git-cliff/Cargo.toml --out dist
+      - name: Upload sdist
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels
+          path: dist
+
+  publish-pypi:
+    name: Publish PyPI packaage
+    runs-on: ubuntu-22.04
+    needs: [pypi-sdist]
```

Comment:
I think this also needs `publish-binaries`

Reply:
Yes, good catch!